### DISCUSSION TITLE
Got any ideas how to bypass Google Antispam?

### DISCUSSION THREAD

**Discussion by Shakura8** [gophish/gophish#3288] [General]
Hi everyone.
Gophish is working great except I'm always getting this "**Warning – phishing (web forgery) suspected**" or the red phishing page on Google after clicking the links on the email campaigns.
I have my own domain and still It's not working well.
With temporary subdomains it's working fine for day or two . 
Any ideas how to tell Google that I'm just doing tests for the employees in my company ? 
Thanks

**Comment by Matt-CAND** [comment]
If the company has its own domain, you can whitelist the phishing email address and this should clear this up. 

But ff they're using Gmail for work email, you're out of luck, basically. That is Gmail working as intended and they will not change this for a single user.
